data/report_templates/project_summary.md

```diff
@@ -17,6 +17,11 @@ NGI Project Name
 NGI Project ID
 :   {{ project.ngi_id }}
 
+{% if project.ngi_facility %}
```

Comment:
Does this create extra line breaks in the Markdown? I found some problems with this in the IGN report - that by having nicely formatted `{% if` statements, I got extra line breaks and it messed up the formatting. Instead, I had to do something like this:

```markdown
{% if project.ngi_facility %}NGI Facility
Genomics {{ project.ngi_facility }} Stockholm{% endif %}
```

But, if tested and it's ok, then keep it as it is for good readability :+1: